src/dmd/templateparamsem.d

```diff
@@ -142,25 +142,49 @@ private extern (C++) final class TemplateParameterSemanticVisitor : Visitor
  */
 RootObject aliasParameterSemantic(Loc loc, Scope* sc, RootObject o, TemplateParameters* parameters)
 {
-    if (o)
+    if (!o)
+        return null;
```

Comment:
Can you split the early return refactoring in a separate in commit in the same PR?

Reply:
done